docs/integrations/sources/mixpanel-migrations.md

```diff
@@ -1,5 +1,63 @@
 # Mixpanel Migration Guide
 
+## Upgrading to 3.0.0
+
+In this release, we introduce breaking change for `CohortMembers` stream: 
+- State is changed to per-partition format.
+- Key is changed to correct unique key (based on 'distinct_id' and 'cohort_id' fields) since previous key was not unique and didn't support possibility for user be in a few different cohorts.
+
+To gracefully handle these changes for your existing connections, we highly recommend resetting your data before resuming your data syncs with the new version.
+
+## Migration Steps
+
+### For Airbyte Open Source: Update the local connector image
+
+Airbyte Open Source users must manually update the connector image in their local registry before proceeding with the migration. To do so:
+
+1. Select **Settings** in the main navbar.
+    1. Select **Sources**.
+2. Find `Mixpanel` in the list of connectors. 
+
+:::note
+You will see two versions listed, the current in-use version and the latest version available.
+::: 
+
+3. Select **Change** to update your OSS version to the latest available version.
+
+### Update the connector version
+
+1. Select **Sources** in the main navbar. 
+2. Select the instance of the connector you wish to upgrade.
+
+:::note
+Each instance of the connector must be updated separately. If you have created multiple instances of a connector, updating one will not affect the others.
+:::
+
+3. Select **Upgrade**
+    1. Follow the prompt to confirm you are ready to upgrade to the new version.
+
+### Refresh affected schemas and reset data
```

Comment:
Do breaking changes affect the schema?

Reply:
In order to update primary key for cohort members client should click on "Refresh Soutce Schema"